Example/Tests/GINICameraViewControllerTests.swift

```diff
@@ -53,32 +53,5 @@ final class CameraViewControllerTests: XCTestCase {
         XCTAssertFalse(cameraViewController.captureButton.isEnabled, "capture button should be disaled when tooltip is shown")
         
     }
-    
-    func testReviewButtonBackgroundBeforeCapturing() {
-        _ = cameraViewController.view
-
-        XCTAssertTrue(cameraViewController.multipageReviewBackgroundView.isHidden,
-                      "multipageReviewBackgroundView should be hidden before capture the first picture")
-        
-    }
-    
-    func testReviewButtonBackgroundAfter1ImageWasCaptured() {
-        _ = cameraViewController.view
-
-        cameraViewController.cameraDidCapture(imageData: imageData, error: nil)
-        XCTAssertTrue(cameraViewController.multipageReviewBackgroundView.isHidden,
-                      "multipageReviewBackgroundView should be hidden after capture the first picture")
-        
-    }
-    
-    func testReviewButtonBackgroundAfter2ImagesWereCaptured() {
```

Comment:
Are these tests really already invalid? It might be interesting to test the integration between getting the event that an image has been captured and seeing the change reflected in the review stack view.

Reply:
That is why I removed them, they should not be part of the unit test since it is the integration between two components.

Reply:
We don't have integration tests yet, IMO is worst to mess up here because we will forget to remove it in the future.